tests/php/sync/test_class.jetpack-sync-queue.php

```diff
@@ -52,13 +52,25 @@ function test_items_exist() {
 	}
 
 	function test_queue_lag() {
-		// lag is the difference in time between the age of the oldest item and the current time
-		$this->queue->reset();
-		$this->queue->add( 'foo' );
-		sleep( 3 );
-		$this->queue->add( 'bar' );
-		sleep( 3 );
-		$this->assertEquals( 6, intval( $this->queue->lag() ) );
+		/* @var $queue Jetpack_Sync_Queue|\PHPUnit\Framework\MockObject\MockObject */
+		$queue = $this->getMockBuilder( 'Jetpack_Sync_Queue' )
+			->setMethods( array( 'generate_option_name_timestamp' ) )
+			->setConstructorArgs( array( 'my_queue' ) )
+			->getMock();
+
+		$queue->expects( $this->at( 0 ) )
+			->method( 'generate_option_name_timestamp' )
+			->will( $this->returnValue( '1527862857.091832' ) );
```

Comment:
since we are hard coding these anyways. Can we hard code them to simpler numbers. So that wn see the difference between them easier.

Reply:
Good call.  Simpler values added in fa7800505.